docs/source/concepts/WorkspaceGroup.rst

```diff
@@ -8,17 +8,36 @@ Workspace Group
 .. contents::
   :local:
 
-A WorkspaceGroup is a group of workspaces.
+A WorkspaceGroup is a group of workspaces. The WorkspaceGroup object does not hold any data itself, but instead holds a list of Workspace objects. They appear as an expandable list of workspaces in the MantidPlot interface (the list of workspaces is also called the ADS or *AnalysisDataService*). Thus, their primary function is to add structure to the ADS and make it more readable.
 
-Most algorithms will execute on a WorkspaceGroup by simply executing the
-algorithm on each workspace contained within.
+Most algorithms can be passed a WorkspaceGroup in place of a normal workspace input, and will simply execute the algorithm on each workspace contained within the group.
 
 Creating a Workspace Group
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Here also it's a little confusing, I think it would be the best to describe each way of group creation next to it's example code.

Reply:
Thanks, I have reorganised the text as per your suggested.